sympy/algebras/quaternion.py

```diff
@@ -459,9 +459,9 @@ def exp(self):
         q = self
         vector_norm = sqrt(q.b**2 + q.c**2 + q.d**2)
         a = exp(q.a) * cos(vector_norm)
-        b = exp(q.a) * sin(vector_norm) * q.b / vector_norm
-        c = exp(q.a) * sin(vector_norm) * q.c / vector_norm
-        d = exp(q.a) * sin(vector_norm) * q.d / vector_norm
+        b = exp(q.a) * sinc(vector_norm).rewrite(sin) * q.b
+        c = exp(q.a) * sinc(vector_norm).rewrite(sin) * q.c
+        d = exp(q.a) * sinc(vector_norm).rewrite(sin) * q.d
```

Comment:
I wonder if it would be better to keep this as `sinc` rather than turn it into a Piecewise...

Reply:
Okay , Then there are more things to change as if we use sinc instead of sin
```
>>> Quaternion(1, 2, 3, 4).exp()
>>> E*cos(sqrt(29)) + 2*E*sinc(sqrt(29))*i + 3*E*sinc(sqrt(29))*j + 4*E*sinc(sqrt(29))*k
```
instead of `E*cos(sqrt(29)) + 2*sqrt(29)*E*sin(sqrt(29))/29*i + 3*sqrt(29)*E*sin(sqrt(29))/29*j + 4*sqrt(29)*E*sin(sqrt(29))/29*k`

Reply:
I think it would be best to discuss the different possibilities on the issue rather than here.

Reply:
I feel we can go ahead with sinc and also change the Docstrings to match the output? @oscarbenjamin